### ISSUE TITLE
Thermal evaporation not requiring thermal energy

### ISSUE THREAD

**Issue by term2112** [mekanism/Mekanism#5832]
*Please use the search functionality before reporting an issue. Also take a look at the closed issues!*

#### Steps to reproduce:
1. build a thermal evap plant
2. feed with either water or brine
3. the brine or lithium is instantly produced

#### Version (make sure you are on the latest version before reporting):
**Forge: 31.0.14** 
**Mekanism: 9.9.7** 

I believe this is an error similar to the bug where the electrolytic separator did not require any energy

**Comment by aidancbrady** [comment]
Another issue- the controller texture is not changing to the active state

**Comment by term2112** [comment]
I have noticed the screen issue too, Is it supposed to turn on once the structure is complete, or when thermal energy is applied?

**Comment by term2112** [comment]
I would also think this would be a good area to implement #5849 if possible with the screen showing the statistics of the machine.

**Comment by pupnewfster** [comment]
The screen is supposed to turn on once the structure is complete. (Which is now fixed in dev again), but the screen is not large enough to show any actual statistics. I will take a look again tomorrow at trying to fix the evap tower not taking heat into the processing speed at which it converts fluids, as after having discussed it some with aidan I now have a few ideas on how to easily fix it.

**Comment by pupnewfster** [comment]
This is fixed in the next version, though the rate shown in the GUI has a few edge case bugs where it will display a value that is very slightly off.

**Comment by pupnewfster** [comment]
Closing this as 9.9.10 is now up and on curseforge.
